templates/includes/sidebar.html

```diff
@@ -1,4 +1,11 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">User: {{ user.username }}</a></li>
+    <li><a href="{% url 'logout' %}">Logout</a></li>
+  {% else %}
+    <a href="{% url 'login' %}">Login</a>
```

Comment:
The login link is correctly set up to use Django's built-in authentication system. Ensure that the login template is available and correctly configured in your project.